### PR TITLE
Support no-compression method in converter

### PR DESCRIPTION
**Pull request by gigony**
If you want to use no compression method for your converted file, you can set the compression method to None in the converter.

In CLI, you can use `--compression` option and set it to any string except `jpeg`.

E.g.,
```
cucim convert --tile-size 512 --overlap 0 --num-workers 12  --output-filename resize.tiff --compression RAW notebooks/input/TUPAC-TR-467.svs
```

The converter will then convert the file without any compression.

Need to update test cases once https://github.com/rapidsai/cucim/pull/433 is merged.

Signed-off-by: Gigon Bae <gbae@nvidia.com>

### CODE REVIEW

File: python/cucim/src/cucim/clara/cli.py [hunk 1]
```diff
@@ -49,12 +49,13 @@ def main():
...
 @click.option('--overlap', type=int, default=0)
 @click.option('--num-workers', type=int, default=os.cpu_count())
+@click.option('--compression', type=str, default='jpeg')
 @click.option('--output-filename', type=str, default='image.tif')
 def convert(src_file, dest_folder, tile_size, overlap, num_workers,
-            output_filename):
+            compression, output_filename):
     """Convert file format"""
     from .converter import tiff
     logging.basicConfig(level=logging.INFO)
```
Comment (grlee77) [inline python/cucim/src/cucim/clara/cli.py:59]:
```suggestion
compression = compression.lower()
if compression in ['raw', 'none']:
    compression = None

```

**Review by grlee77 — COMMENTED** [review]
Added a minor suggestion to make the compression string case insensitive and raise an error on anything other than 'jpeg', 'raw' or 'none'.

File: python/cucim/src/cucim/clara/converter/tiff.py [hunk 2]
```diff
@@ -65,8 +66,14 @@ def svs2tif(input_file, output_folder, tile_size, overlap,
     logger.info("        tile size: %d", tile_size)
     logger.info("          overlap: %d", overlap)
     logger.info("      num_workers: %d", num_workers)
+    logger.info("      compression: %s", compression)
     logger.info("  output filename: %s", output_filename)
 
+    if compression == "jpeg":
+        compression = ("jpeg", 95)
+    else:  # handles only jpeg, othwerwise no compression
+        compression = None
```
[the remark this answers is not in the document]
Comment (gigony) [reply python/cucim/src/cucim/clara/converter/tiff.py]: Thanks @grlee77  for the suggestion! Updated code.

File: python/cucim/src/cucim/clara/cli.py (hunk 1 above)
Comment (gigony) [reply python/cucim/src/cucim/clara/cli.py:59]: Thanks for the suggestion!

**Review by grlee77 — APPROVED** [review]
This looks good to me, thanks

**Comment by grlee77** [comment]
@gpucibot merge

**Comment by gigony** [comment]
Thanks a lot @grlee77 for handling/merging this and #433 !

**Pull request merged**